frontend/src/components/PsyListing/PsyListing.jsx

```diff
@@ -69,18 +87,46 @@ const PsyListing = () => {
     }
   };
 
+  const success = pos => {
+    const { longitude, latitude } = pos.coords;
+    setCoords({ longitude, latitude });
+    setGeoStatus(geoStatusEnum.GRANTED);
```

Comment:
it's ok here, but note that this will do 2 re renders, if you have two strongly linked state you should pu them in one object

Reply:
Thanks for the advice :)